service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -99,4 +103,45 @@ public SystemStatusSystems status() {
     }
     return result;
   }
+
+  EntitiesApi entitiesApi(AuthenticatedUserRequest user) {
+    return new EntitiesApi(getApiClient(user));
+  }
+
+  public void exportDataRepoDataset(
+      AuthenticatedUserRequest user, String snapshotId, String workspaceId) {
+    // todo
+  }
+
+  public void exportWorkspaceDataset(
+      AuthenticatedUserRequest user, String workspaceIdSource, String workspaceId) {
+    try {
+      // build source name
+      WorkspaceDetails workspaceDetailsSource =
+          workspacesApi(user).getWorkspaceById(workspaceIdSource, ACCESS_LEVEL).getWorkspace();
+      WorkspaceName workspaceNameSource =
+          new WorkspaceName()
+              .namespace(workspaceDetailsSource.getNamespace())
+              .name(workspaceDetailsSource.getName());
+
+      // build destination name
+      WorkspaceDetails workspaceDetailsDest =
+          workspacesApi(user).getWorkspaceById(workspaceId, ACCESS_LEVEL).getWorkspace();
+      WorkspaceName workspaceNameDest =
+          new WorkspaceName()
+              .namespace(workspaceDetailsDest.getNamespace())
+              .name(workspaceDetailsDest.getName());
+
+      // possible bug: empty entityType and entityNames copies all entities
+      EntityCopyDefinition body =
+          new EntityCopyDefinition()
+              .sourceWorkspace(workspaceNameSource)
+              .destinationWorkspace(workspaceNameDest)
+              .entityType("")
+              .entityNames(List.of());
+      entitiesApi(user).copyEntities(body, false);
+    } catch (ApiException e) {
+      // todo
```

Comment:
What should we do if we get an error? There are many errors that are possible, but one obvious one is what happens if you export to workspace with entities that already exist in the destination workspace?